scripts/integer-tests.sh

```diff
@@ -9,13 +9,15 @@ function usage() {
     echo "--rust-toolchain          The toolchain to run the tests with default: stable"
     echo "--multi-bit               Run multi-bit tests only: default off"
     echo "--cargo-profile           The cargo profile used to build tests"
+    echo "--tfhe-package            The package spec like tfhe@0.4.2, default=tfhe"
```

Comment:
Unrelated but could be fixed here, line 6 says shortint instead of integer

Reply:
yep will do